core/vm/evm.go

```diff
@@ -201,6 +201,13 @@ func (self *EVM) Main(tx *Transaction, opts ExecutionOpts) (ret ExecutionResult)
 	}
 	gas_left := gas_cap
 	contract_creation := self.trx.To == nil
+
+	gas_intrinsic, err := IntrinsicGas(self.trx.Input, contract_creation, self.rules.IsHomestead)
```

Comment:
This is a bug introduced by Matus in master branch that you are based on. I did remove it from master by force push so it is showing here as your new change. Remove it...